configs/components/marionette-collective.rb

```diff
@@ -66,7 +66,33 @@
   when "aix"
     pkg.install_service "resources/aix/mcollective.service", nil, "mcollective"
   when "windows"
-    puts "Service files not enabled on windows"
+    service_hash = [{
+      :directory_ref => settings[:bindir_id],
+      :id => "MCOService",
+      :guid => "7601FCEA-90B3-CC69-6A69-4087FBC7292D",
+      :win64 => settings[:win64],
+      :file => {
+        :id => "RubyWExe",
+        :source => settings[:rubyw],
+      },
+      # This service is installed with start set to demand because
+      # it won't be correctly configured to start right away. The
+      # puppet run that configures mcollective will allow it to start
+      # and set it to automatic
+      :serviceinstall => {
+        :id => "MCOServiceInstaller",
+        :description => "Puppet Labs server orchestration framework",
+        :displayname => "Marionette Collective Server",
+        :name => "mcollective",
+        :start => "demand",
+        :arguments => '-I&quot;[INSTALLDIR]\\lib&quot; -rubygems &quot;[INSTALLDIR]\\bin\\mcollectived&quot; --daemonize',
```

Comment:
Soo, isn't that the point of this question? What is the directory base we want to give users the ability to rip out and replace? `C:\Program Files\Puppet Labs` or `C:\Program Files\Puppet Labs\puppet`. We've traditionally had this be `C:\Program Files\Puppet Labs\Puppet`. Can I change the default on people? Will people have automation expecting to be able to set it to `/some/randome/place/bin/facter.exe` and then be surprised that it's actually `/some/randome/place/puppet/bin/facter.exe`? I know it's a required property, I know the user can change it, I know it gets set in the main wxs file. What's the path I should have after `[INSTALLDIR]`? `\\lib` or `\\puppet\\lib`?